my_game.py

```diff
@@ -460,13 +480,13 @@ def on_show_view(self):
             scale=CONFIG['SPRITE_SCALING']
         )
 
-        # Spawn Asteroids
-        for r in range(CONFIG['ASTEROIDS_PR_LEVEL']):
-            self.asteroid_list.append(Asteroid())
-
         # setup spawn_ufo to run regularly
         arcade.schedule(self.spawn_ufo, CONFIG['UFO_SPAWN_RATE'])
 
+        # Start level 1
+        self.next_level(1)
```

Comment:
In a sense we could also set level to 0 initially - so we could just call _next_level_ from the start. And thus remove a couble of lines... but ill leave it to you to decide wether its worth it

Reply:
We need to call it here, since must populate the asteroids sprite list.

Reply:
Or, do you mean, that since the asteroids list would initially be empty, it would call next_level() and populate the asteroids list? I guess you are right about that.

Reply:
We could keep it like this, and then add a config for setting the level the game begins at. I guess that could be our difficulty setting?

Reply:
yea, that sounds good - starting at a specific level seems like a good setting. But for the difficulty i think we should also have a variable for how much the difficulty increases per level (both in asteroid speed number, same for ufos - and stuff like that)

But either way thats a whole nother feature